Trouble shooting guide 2500

rlcheng opened the pull request:
| | |
|:---|:---|
|**_Related Issue(s)_**| 2500 |
|**_Has Unit Tests (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

Fix broken links in INSTALL.md to point to troubleshooting.md

## Rationale

Links broken.

## Testing/Review Recommendations

## Future Work

rlcheng committed:
trouble-shooting-guide-2500
 - Add troubleshooting.md.
   This is currently a copy of the troubleshooting section of
   https://nasa.github.io/fprime/INSTALL.html

rlcheng committed:
trouble-shooting-guide-2500
 - Update INSTALL.md and remove troubleshooting guide and redirect
   to newly created separate file.
 - Remove empty line / space from troubleshooting.md.

rlcheng committed:
trouble-shooting-guide-2500
 - Update section links in INSTALL.md System Requirements to point
   to troubleshooting.md.

rlcheng committed:
Merge branch 'nasa:devel' into trouble-shooting-guide-2500

LeStarch commented:
@rlcheng we should add "debian" to the expect list.  If it is there, make sure it is lower case (e.g. `debian`)

rlcheng committed:
trouble-shooting-guide-2500
 - Update expect.txt for text 'debian'. Changed from 'Debian' to
   catch all expected spelling.
